sys/aarch64/gpio.c

```diff
@@ -0,0 +1,94 @@
+#include <sys/rman.h>
+#include <sys/bus.h>
+#include <aarch64/bcm2835reg.h>
+#include <aarch64/bcm2835_gpioreg.h>
+
+/* The function select registers are used to define the operation of the
+ * general-purpose I/O pins. */
+#define GPFSEL(x) ((x) * sizeof(uint32_t))
+
+#define GPFSEL_PINS_PER_REGISTER 10
+#define GPFSEL_BITS_PER_PIN 3
+
+/* The output set registers are used to set a GPIO pin. */
+#define GPSET(x) (0x1C + (x) * sizeof(uint32_t))
```

Comment:
You have corresponding definitions here and in `bcm2835_gpioreg.h`. Please sort it out.

Reply:
Done.